lib/Services/Ratings.php

```diff
@@ -32,12 +32,13 @@ class Ratings
     private $metadataFieldCount;
 
     public function __construct(
-        ApiClientInterface $client,
+        AbstractApiClient $client,
         $vmId,
         $metadataFieldAverage,
         $metadataFieldCount
     ) {
         $this->client = $client;
+        $this->client->disableCaching();
```

Comment:
Any additional calls after instantiating a RatingService made by the same client will have caching disabled. 

Is the intended way to obtain a rating service to normally first construct a separate client for it?